hs/WebParsing/TimeTableParser.hs

```diff
@@ -17,15 +18,111 @@ import WebParsing.TimeConverter
 import Control.Monad.IO.Class (liftIO, MonadIO)
 import Control.Monad.Trans.Reader (ReaderT)
 import Config (databasePath, timetableUrl)
+import Database.Tables (Department(Department))
 
 parseTT :: IO ()
 parseTT = do
     rsp <- simpleHTTP (getRequest timetableUrl)
     body <- getResponseBody rsp
+
     let depts = getDeptList $ parseTags body
+    let deptCode = getDeptCode $ parseTags body
+    let deptName = getDeptName $ parseTags body
```

Comment:
You can remove this `let` and the one above it. See another place in the same PR where these are removed [here](https://github.com/Courseography/courseography/pull/754/files#diff-51b14c0659ad1c06571c3bf833a68171R67).